parser/parse.go

```diff
@@ -127,7 +136,7 @@ func (b *Builder) importBuildPackage(dir string) (*build.Package, error) {
 	buildPkg, err := b.importWithMode(dir, build.ImportComment)
 	if err != nil {
 		if _, ok := err.(*build.NoGoError); !ok {
-			return nil, fmt.Errorf("unable to import %q: %v", dir, err)
+			return nil, &importError{dir: dir, err: err}
```

Comment:
would rather follow https://blog.golang.org/go1.13-errors with error wrapping with `%w`.

Note that the type cast below would not work if you return a pointer here.

Reply:
The specific error we're looking for isn't a concrete type: https://github.com/golang/go/blob/master/src/go/build/build.go#L739 so wrapping won't work unless we wrap our custom type.

Reply:
Couldn't you check for `*build.NoGoError`?

Reply:
Oh, `*build.NoGoError` is what you want log output for? Am confused.

Reply:
`*build.NoGoError` is [already ignored](https://github.com/kubernetes/gengo/blob/ae764952c2a6af9c1b90ab38c9564b29623962ad/parser/parse.go#L138) in the `importBuildPackage` function so it's not returned.